package.json

```diff
@@ -49,11 +49,11 @@
     "@ngrx/router-store": "~10.0.0",
     "@ngrx/store": "~10.0.0",
     "@nguniversal/express-engine": "^10.1.0",
-    "@spartacus/assets": "3.0.0-rc.0",
-    "@spartacus/core": "3.0.0-rc.0",
-    "@spartacus/misc": "^3.0.0-rc.0",
-    "@spartacus/storefront": "3.0.0-rc.0",
-    "@spartacus/styles": "3.0.0-rc.0",
+    "@spartacus/assets": "3.0.0",
```

Comment:
Aligned with @giansr to update dependencies to final 3.0.0 SPA versions as rc4 was the last release candidate.
In final version, the issue with trying to fetch cart for anonymous user (when customer is logged in) is resolved.

Reply:
Please update also fsastorefrontlib package.json. There we have peer dependency on @spartacus/core.